--- a/node_modules/strip-json-comments/index.js	2016-02-10 00:06:26.480195656 +0100
+++ b/node_modules/strip-json-comments/index.js	2016-02-10 00:07:38.499187621 +0100
@@ -35,8 +35,13 @@
 		if (insideString) {
 			continue;
 		}
-
-		if (!insideComment && currentChar + nextChar === '//') {
+		
+		if (!insideComment && currentChar + nextChar === '!!') {
+                        ret += str.slice(offset, i);
+                        offset = i;
+                        insideComment = singleComment;
+                        i++;
+                } else if (!insideComment && currentChar + nextChar === '//') {
 			ret += str.slice(offset, i);
 			offset = i;
 			insideComment = singleComment;
